CHANGELOG.md

```diff
@@ -2,13 +2,8 @@
 
 ## 1.1.0 (2023-01-24)
 
-### Behavior Changes
-- `SnowparkSourceCodeExtractionException` is raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
--  In `Session.create_dataframe()`, if `data` is a pandas DataFrame and the optional parameter `schema` is specified, `Session.create_dataframe()` now uses `schema` as the resulting DataFrame's schema.
```

Comment:
@sfc-gh-stan I remember you said a bug was fixed. Was it also rolled back with this revert?

Reply:
Yes.